public-ui/src/app/data-browser/views/error-message/error-message-react.component.tsx

```diff
@@ -0,0 +1,55 @@
+import {
+  Component,
+  Input,
+  ViewEncapsulation
+} from '@angular/core';
+import { BaseReactWrapper } from 'app/data-browser/base-react/base-react.wrapper';
+import { ClrIcon } from 'app/utils/clr-icon';
+import * as React from 'react';
+
+interface Props {
+    dataType: string;
+}
+
+export class ErrorMessageReactComponent extends React.Component<Props, {}> {
+
+  constructor(props: Props) {
+    super(props);
+  }
+
+  render() {
+    const {dataType} = this.props;
+    const iconClass = 'alert-icon';
+    const iconShape = 'exclamation-triangle';
+    const alertText =  (dataType === 'data') ? '<span className="alert-text"></span>' : '<span className="alert-text">' +
+                                'Sorry, the chart cannot be displayed. Please try refreshing the page.' +
+                              '</span>';
+    return <div className='alert alert-box'>
+      <div className='alert-items'>
+        <div className='alert-item static'>
+          <div className='alert-icon-wrapper'>
+            <ClrIcon shape={iconShape} className={iconClass}
+                            style={{width: 26, height: 26}} />
+          </div>
+          <span className='alert-text'>
+          {dataType === 'data' ? 'Sorry, the data are currently unavailable. Please try refreshing the page or returning home <a [routerLink]="[\'\']">here.</a>'
```

Comment:
I realize that currently, the `dataType` will never be 'data' for this component, but we shouldn't leave `[routerLink]="[\'\']"` here.

Reply:
So, the datatype would be data from here https://github.com/all-of-us/data-browser/blob/master/public-ui/src/app/data-browser/views/ehr-view/ehr-view.component.html#L32. 

I am assuming we would have to wait till we figure out react router to place home link in here.